bundles/org.openhab.binding.tesla/README.md

```diff
@@ -182,9 +182,13 @@ Additionally, these advanced channels are available (not all are available on al
 | shiftstate                | String                   | Shift State                   | Indicates the state of the transmission, “P”, “D”, “R”, or “N”                                                   |
 | sidemirrorheaters         | Switch                   | Side Mirror Heaters           | Indicates if the side mirror heaters are switched on                                                             |
 | smartpreconditioning      | Switch                   | Smart Preconditioning         | Indicates if smart preconditioning is switched on                                                                |
+| softwareupdateavailable   | Switch                   | Update Available              | Car Software update available, automatically generated comparing "version" to "update version"                   |
+| softwareupdatestatus      | String                   | Update Status                 | Car Software update status, e.g. "downloading_wifi_wait", "installing"                                           |
+| softwareupdateversion     | String                   | Update version                | Car software version to update to, e.g. "2023.32.9" or empty                                                     |
+| softwareversion           | String                   | Software version              | Software version, e.g. "2023.32.6 93d75c736e0c"                                                                  |
```

Comment:
This is usually modelled as thing property, not as a channel. What is the use case of moddeling this as a channel?

Reply:
As far as I am aware, I cannot bind a property to an item, and therefore cannot create any rules to act on that.

The main reason is that I want OH to be able to notify me of any pending updates.

Reply:
The comment was meant for `softwareversion` line 188.

I can undesrstand you want `softwareupdateversion` as channel

Reply:
Sure, but this pull request is about exposing the current version as well as a channel. So that it can be bound to an item and have rules act on that. And for this, having the current version is important as well.

Mainly an quality-of-life enhancement for people who don't want or are not able to interact directly with the API via http calls.

Reply:
If,  like you mentioned before, the rule is meant to notify you on about an update, `softwareupdateversion` should is  enough. As that only holds a value if there is an update. You don't need the current software at all, or am i missing something?

I can understand quality of life enhancements, there is also some balancing with maintainability / duplicate info etc, so just trying to understand the use case.

Reply:
Actually, yes, you are correct. Let me put back the previous state in which this was only a property of the thing.

I will be at my keyboard to do that in another hour so.

Reply:
Done, "softwareversion" is a property called "version" once more.